src/app-modules/docker-compose

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+set -x
+
+docker_compose_cmd=docker-compose
+docker_cmd=docker
+
+app_load() {
+    local url="$1"
+    local input_file="$2"
+
+    $docker_cmd image load < "$input_file"
+}
+
+app_save() {
+    local url="$1"
+    local output_file="$2"
+
+    $docker_cmd image save "$url" -o "$output_file"
+}
+
+app_rollout() {
+    local manifests_dir="$1"
```

Comment:
This is wrong, should be `$2` according to the API docs.

Reply:
fixed.